integration-testing/src/infra/mod.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! Setup infra for integration testing
+
+pub mod subnet;
+pub mod util;
+
+use ipc_sdk::subnet_id::SubnetID;
+use std::sync::atomic::{AtomicU16, Ordering};
+use std::sync::Arc;
+
+const DEFAULT_IPC_AGENT_URL: &str = "http://localhost:3030/json_rpc";
+const DEFAULT_NODE_API_BASE_PORT: u16 = 1230;
+const DEFAULT_MIN_STAKE: f64 = 1.0;
+
+pub struct SubnetTopology {
```

Comment:
Do you mind adding comments here so it is clear what each field of the struct is?